### DISCUSSION TITLE
Using with Test Server

### DISCUSSION THREAD

**Discussion by yongra** [tl-its-umich-edu/my-learning-analytics#1306] [General]
Can MyLA set up with the test Canvas server?

**Comment by jonespm** [comment]
It can be setup. But the issue is that Canvas only generates data exports from production. So any content created in test won't be reflected in MyLA, it would always be a few weeks behind.

It can similarly be used in beta which would be only a week off but would still have to use production data. MyLA doesn't pull from the Canvas APIs, just from the Canvas Data exports in the Data Warehouse.

**Reply by jonespm** [reply]
We were watching Canvas Data 2 but I don't believe that they were going to be supporting test or beta for that either. They'd said

> Customers will also be able to view the beta version of Canvas Data 2 schema, which allows customers to view new changes prior to the changes being released to Canvas production. This behavior is not to be confused with accessing Canvas Data 2 directly in the beta environment, which will not be supported.

**Reply by yongra** [reply]
Thank you. I am setting it up to try to make sure it is working first. So, seeing the old data is not a problem. As long as it works, I will change the setting to the production.
